lib/json_api_resource/resource.rb

```diff
@@ -1,3 +1,12 @@
+require 'active_support'
+require 'active_support/concern'
+require 'active_support/core_ext/module'
+require 'active_support/core_ext/class/attribute'
+require 'active_model'
+require 'active_model/model'
+require 'active_model/validations'
+require 'active_model/callbacks'
+
 module JsonApiResource
   class Resource
     include ActiveModel::Model
```

Comment:
I think that's probably fine. I don't know of a usecase for this kind of object outside of a web framework.